sites/svelte.dev/src/routes/banner.json/+server.js

```diff
@@ -0,0 +1,27 @@
+import { json } from '@sveltejs/kit';
+
+export const GET = async () => {
+	return json(
+		/** @satisfies {Awaited<ReturnType<typeof import('@sveltejs/site-kit/components').fetchBanner>>} */ ([
+			{
+				id: 'advent2023',
+				start: new Date('1 Dec, 2023 00:00:00 UTC'),
+				end: new Date('24 Dec, 2024 23:59:59 UTC'),
+				arrow: true,
+				content: 'Svelte Advent 2023 is here!',
+				href: 'https://advent.sveltesociety.dev/'
+			},
+
+			// This one skips the blog post and just changes the link
+			{
+				id: 'after-runes',
+				start: new Date('25 Dec, 2023 00:00:00 UTC'),
+				end: new Date('15 Jan, 2024 23:59:59 UTC'),
```

Comment:
```suggestion
				end: new Date('1 Jan, 2024 00:00:00 UTC'),
```

Reply:
Would be nice to show it for a few days after its over 😅

5 Jan should be good. +10 days after it should finish

Reply:
The text says "see you next year" and it's not that important to link to advent of Svelte after it's over, so I'm in favor of ending on Jan 1st